src/tic_tac_toe_pair/console.clj

```diff
@@ -1,43 +1,99 @@
 (ns tic-tac-toe-pair.console 
-  (:require [clojure.string :as s])
+  (:require [clojure.string :as s]
+            [tic-tac-toe-pair.board :refer :all]
+            [tic-tac-toe-pair.rules :refer :all])
   (:gen-class))
 
 (def board-shape 
-  (str "     |     |     \n"
-       "  0  |  1  |  2  \n"
-       "     |     |     \n"
-       "-----------------\n"
-       "     |     |     \n"
-       "  3  |  4  |  5  \n"
-       "     |     |     \n"
-       "-----------------\n"
-       "     |     |     \n"
-       "  6  |  7  |  8  \n"
-       "     |     |     "))
+  (str "          |     |     \n"
+       "       0  |  1  |  2  \n"
+       "          |     |     \n"
+       "     -----------------\n"
+       "          |     |     \n"
+       "       3  |  4  |  5  \n"
+       "          |     |     \n"
+       "     -----------------\n"
+       "          |     |     \n"
+       "       6  |  7  |  8  \n"
+       "          |     |     \n"))
 
 (defn keyword-to-token [keyword] (s/upper-case (name keyword)))
 
-(defn build-square [board square]
+(defn set-square-content [board square]
   (let [token (get board square)]
     (if (nil? token)
       " "
       (keyword-to-token token))))
 
+(defn replace-square-position [board-string board square-pos]
+  (s/replace board-string (str square-pos) (set-square-content board square-pos)))
+
 (defn build-board [board]
-  (loop [square         0
+  (loop [square-pos     0
          filled-board   board-shape]
-    (if (>= square (count board))
+    (if (>= square-pos (count board))
       filled-board
-      (recur
-        (inc square)
-        (s/replace
-          filled-board
-          (str square)
-          (build-square board square))))))
+      (recur (inc square-pos) (replace-square-position filled-board board square-pos)))))
 
 (defn draw-board [board] 
   (println (build-board board)))
 
-(defn get-move-location [] 
-  (println "Enter a move:")
-  (Integer/parseInt (read-line)))
\ No newline at end of file
+(defn clear-terminal []
+  (do (print "\u001b[2J") (print "\u001B[0;0f")))
+
+(defn draw-header []
+  (println "---------------------------\nTic Tac Toe\n---------------------------"))
+
+(defn draw-player-info []
+  (println "Player 1 (X)     Player 2 (O)\n"))
```

Comment:
This method knows a lot about the player implementation. It would be better to refactor this so that the method simply takes in some input and prints that.

Reply:
Addressed `draw-player-info` method updates.